_studio/mfx_lib/CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ include_directories( ${MSDK_STUDIO_ROOT}/shared/asc/include )
 include_directories( ${MSDK_STUDIO_ROOT}/hevce_hw/h265/include )
 
 list( APPEND cdirs
-  brc h264_enc mpeg2_dec  vc1_dec vc1_common
+  brc h264_enc mpeg2_dec vc1_dec vp9_dec vc1_common
```

Comment:
why there is no h264_dec and h265_dec in this list? and if we don't need these, why we need vc1 and vc9

Reply:
Unfortunately all of them currently required for compilation. Refactoring out of scope for this PR.